src/App.js

```diff
@@ -1,27 +1,14 @@
 import React from 'react';
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
-  Redirect,
-} from 'react-router-dom';
 import './App.css';
 import { ApolloProvider } from '@apollo/client';
 
 import apolloClient from './apollo';
-import LandingPage from './landing/LandingPage';
-import NotFoundPage from './NotFound';
+import MyRouter from './RouterLogic';
```

Comment:
nitpick on name: Typically we don't use "My" prefix in a real app. Let's treat it as such. Try to choose a more appropriate name.

Reply:
updated the name to AppRouter